README.md

```diff
@@ -123,34 +129,101 @@ module "us_east_1b_private_subnets" {
-| `private_network_acl_egress`  | see [variables.tf]    | An Egress ACL which will be added to the new Private Network ACL                                                                                                                          |    No    |
-| `private_network_acl_ingress` | see [variables.tf]    | An Ingress ACL which will be added to the new Private Network ACL                                                                                                                         |    No    |
+| `public_network_acl_egress`   | see [variables.tf](https://github.com/cloudposse/terraform-aws-named-subnets/blob/master/variables.tf)    | Egress rules which will be added to the new Public Network ACL                                        |    No    |
+| `public_network_acl_ingress`  | see [variables.tf](https://github.com/cloudposse/terraform-aws-named-subnets/blob/master/variables.tf)    | Ingress rules which will be added to the new Public Network ACL                                       |    No    |
+| `private_network_acl_egress`  | see [variables.tf](https://github.com/cloudposse/terraform-aws-named-subnets/blob/master/variables.tf)    | Egress rules which will be added to the new Private Network ACL                                       |    No    |
+| `private_network_acl_ingress` | see [variables.tf](https://github.com/cloudposse/terraform-aws-named-subnets/blob/master/variables.tf)    | Ingress rules which will be added to the new Private Network ACL                                      |    No    |
 
 
 ## Outputs
 
-| Name            | Description                                  |
-|:----------------|:---------------------------------------------|
-| ngw_id          | ID of NAT Gateway                            |
-| ngw_private_ip  | A private IP address of a NAT Gateway        |
-| ngw_public_ip   | A public IP address of a NAT Gateway         |
-| subnet_ids      | IDs of Subnets                               |
-| route_table_ids | IDs of Route Tables                          |
+| Name                      | Description                                  |
+|:--------------------------|:---------------------------------------------|
+| ngw_id                    | NAT Gateway ID                               |
+| ngw_private_ip            | Private IP address of the NAT Gateway        |
+| ngw_public_ip             | Public IP address of the NAT Gateway         |
+| route_table_ids           | Route Table IDs                              |
+| subnet_ids                | Subnet IDs                                   |
+| subnet_names_subnet_ids   | Map of subnet names to subnet IDs            |
+
+
+Given the following configuration
+
+```hcl
+locals {
+  public_cidr_block  = "${cidrsubnet(var.vpc_cidr, 1, 0)}"
+  private_cidr_block = "${cidrsubnet(var.vpc_cidr, 1, 1)}"
+}
+
+module "public_subnets" {
```

Comment:
Add zone1 or something to the name to convey that to create distributed zones the module should be instaniated multiple times.

Reply:
Also, add to read me how to architect for HA